Sources/SwiftDocCUtilities/ArgumentParsing/ActionExtensions/ConvertAction+CommandInitialization.swift

```diff
@@ -87,8 +87,30 @@ extension ConvertAction {
 
 private func symbolGraphFiles(in directory: URL?) -> [URL] {
     guard let directory = directory else { return [] }
+    guard let enumerator = FileManager.default.enumerator(
```

Comment:
Yeah agreed- I mainly did this to remove the `appendingPathComponent($0)` since I was doing the audit for calls to that missing the `isDirectory`. I think it makes sense to update to the more modern URL-based API here regardless but maybe we should skip this on `release/5.7`...